contracts/Cashier.sol

```diff
@@ -178,7 +178,10 @@ contract Cashier is ICashier, UsingHelpers, ReentrancyGuard, Ownable, Pausable {
         (
             voucherDetails.currStatus.status,
             voucherDetails.currStatus.isPaymentReleased,
-            voucherDetails.currStatus.isDepositsReleased
+            voucherDetails.currStatus.isDepositsReleased,
+            
```

Comment:
Please remove the 2 empty lines and the line with the sole comma.

Reply:
The comma can't be removed. It is separating placeholder space for two return values that are not being used. It is customary to use  a comma with a space for each return value not used. I reduced the space even though this makes it less obvious that these are places holders. The comma cannot be removed because then the number of return values doesn't match the function.

Reply:
OK, I didn't notice that purpose.